proposals/stage-2/CPC_CHARTER/CPC-CHARTER.md

```diff
@@ -247,8 +247,8 @@ processes for filling this seat will be as follows:
 
 * As a constellation member of the OpenJS Foundation, the Node.js project
   will elect one Board representative from either the TSC or
```

Comment:
Suggestion that resolves: https://github.com/openjs-foundation/bootstrap/pull/127/files#r268289608

```suggestion
  will elect one Board representative from either the Technical Steering Committee (TSC) or
```